service_implementation/advanced_plm.md

```diff
@@ -1 +1,455 @@
-# Beyond just files: Using PLM for versioning and metadata
\ No newline at end of file
+# Beyond just files: Using PLM for versioning and metadata
+This document describes how to use PLM services from _inside_ a service, i.e., by calling the PLM API. The text includes  example code.
+There are two types of PLM services for use in CAxMan; both are described below:
+1) those that are integrated in the GSS framework;
+2) the PLM Server native web-services.
+
+## GSS
+
+The PLM Server has its own implementation of the GSS services. GSS offers SOAP and REST services:
+- WSDL (SOAP): https://caxman.clesgo.net/jotne/GSS/FileWebService?wsdl
+- REST end point: https://caxman.clesgo.net/jotne/GSS/resources/plm
+
+GSS covers a small subset of simplified methods (services) of the PLM Server.
+
+Java (Maven) client based examples can be retrieved from here: https://github.com/CAxMan/PlmGSS-samples
+
+The PLM Server is also accessible via the GSS implementation by SINTEF.
+Code samples of that are here: https://github.com/CAxMan/infrastructureClients/tree/master/gssClients
+
+The following GSS methods (SOAP) are only implemented for the PLM Server storage option, not for the SWIFT storage:

```

Comment:
Again, I'm uncomfortable with the wording "GSS methods". These SOAP methods are _not_ GSS methods but methods of the PLM API.